pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/QuestionDao.java

```diff
@@ -1717,4 +1735,14 @@ private CompositeQuestionDef buildCompositeQuestionDef(CompositeQuestionDto dto,
 
         return builder.build();
     }
+
+    private long findMimeTypeIdOrInsert(String mimeType) {
```

Comment:
You can put this in `JdbiFileQuestion` as a `default` method.

Reply:
I was trying to make it default initially, but it not allows combination "private default". But I wasn't going to make this method available to other classes.

Reply:
Why not make it available? If someone wants to "find or insert" a mime type (e.g. in a unit test) then they'll have to write their own?

Reply:
ok, did it 'default'